test/parallel/test-buffer.js

```diff
@@ -1222,11 +1222,11 @@ assert.throws(function() {
 
 assert.throws(function() {
   new Buffer();
-}, /must start with number, buffer, array or string/);
+}, /must start with number, buffer, array or string/i);
 
 assert.throws(function() {
   new Buffer(null);
-}, /must start with number, buffer, array or string/);
+}, /must start with number, buffer, array or string/i);
```

Comment:
I made the regex case insensitive because if that were already the case, this test wouldn't have broken at all.